test/test-lists.ts

```diff
@@ -59,7 +59,8 @@ const bypass = new Set([
     "cloud.dbank.com",
     "bybi75-alternate.app.link",
     "mz4t6.rdtk.io",
-    "ether.fi" // https://x.com/ether_fi/status/1838643492102283571
+    "ether.fi", // https://x.com/ether_fi/status/1838643492102283571
+    "staktrustwallet.epizy.com"
```

Comment:
Sorry, can you remove this line now since you are instead applying it to custom-tlds